src/test/java/teammates/test/cases/action/InstructorFeedbackQuestionEditActionTest.java

```diff
@@ -1468,6 +1469,141 @@ public void testExecuteAndPostProcessRubricQuestion() {
         FeedbackSessionsLogic.inst().deleteFeedbackSessionCascade(fs.getFeedbackSessionName(), fs.getCourseId());
     }
 
+    @Test
+    public void testExecuteAndPostProcessRankQuestion() {
```

Comment:
The naming convention is `featureUnderTest_testScenario_expectedBehavior()`. Also split up the different cases into different tests.

https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html